docs/src/administration/users.md

```diff
@@ -104,9 +104,9 @@ title=In the Console
 - Select the project where you want to add a new user.
 - Click {{< icon settings >}} **Settings**.
 - Click **Access**.
-- Expand the user you want to delete.
-- Click **Delete**.
-- Click **Save**.
+- Click the user you want to delete.
+- Click **Remove user**.
+- Click **Accept**.
 
 <--->
```

Comment:
🛠️ For consistency's sake, I would suggest also changing the terminology from _delete_ to _remove_ in the rest of the section as well (**Using the CLI tab** + the 2 sentences at the end).